<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallsWidget</class>
 <widget class="QWidget" name="BallsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="ballColor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="AddGraphicsView" name="tableView"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ballColorLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Цвет шара</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="prevButton">
     <property name="text">
      <string>&lt; Стол</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Удар &gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>158</width>
       <height>112</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="tip">
     <property name="text">
      <string>Нажмите правой клавишей по шару для выбора</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Щелкните 2 раза по столу для добавления шара</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">gui/objects/addgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
